src/textUtils.c

```diff
@@ -59,39 +59,70 @@ size_t str_formatAdaAmount(uint64_t amount, char* out, size_t outSize)
 	return rawSize + suffixLength;
 }
 
-size_t str_formatUint64(uint64_t number, char* out, size_t outSize)
+static size_t stringifyUint64ToBufferReverse(uint64_t number, char* ptr, char* end)
```

Comment:
Maybe more meaningful names for ptr and end so that it is clear what they mean?

Reply:
The WRITE_CHAR macro itself calls them ptr and end, and end is pretty self explanatory I think. I renamed ptr to currChar.